ext/pdo_firebird/config.m4

```diff
@@ -28,13 +30,7 @@ if test "$PHP_PDO_FIREBIRD" != "no"; then
 
     PHP_CHECK_LIBRARY([fbclient], [isc_detach_database],
```

Comment:
If we want to stick with not checking for headers, we could likely check for the symbol `fb_get_master_interface`. I guess that would require an additional `PHP_CHECK_LIBRARY`. Or is there an "abbreviation"?

Reply:
This one is a nice find. It was added in Firebird 3.0.0. So when for some reason fb_config isn't found on the system, the fallback check in config.m4 can be this then:

```diff
-    PHP_CHECK_LIBRARY([fbclient], [isc_detach_database],
-      [FIREBIRD_LIBNAME=fbclient],
-      [PHP_CHECK_LIBRARY([gds], [isc_detach_database],
-        [FIREBIRD_LIBNAME=gds],
-        [PHP_CHECK_LIBRARY([ib_util], [isc_detach_database],
-          [FIREBIRD_LIBNAME=ib_util],
-          [AC_MSG_FAILURE([libfbclient, libgds or libib_util not found.])],
-          [$FIREBIRD_LIBDIR_FLAG])],
-        [$FIREBIRD_LIBDIR_FLAG])],
+    PHP_CHECK_LIBRARY([fbclient], [fb_get_master_interface],
+      [],
+      [AC_MSG_FAILURE([libfbclient, libgds or libib_util not found.])],
       [$FIREBIRD_LIBDIR_FLAG])
-    PHP_ADD_LIBRARY_WITH_PATH([$FIREBIRD_LIBNAME],
+    PHP_ADD_LIBRARY_WITH_PATH([fbclient],
       [$FIREBIRD_LIBDIR],
       [PDO_FIREBIRD_SHARED_LIBADD])
     PHP_ADD_INCLUDE([$FIREBIRD_INCDIR])
```

Reply:
Ah, right, of course no more need to check for `isc_detach_database()` at all. Also nice to have another variable removed. Thank you!